include/af/defines.h

```diff
@@ -424,12 +424,16 @@ typedef enum {
 ////////////////////////////////////////////////////////////////////////////////
 typedef enum {
     AF_COLORMAP_DEFAULT = 0,    ///< Default grayscale map
-    AF_COLORMAP_SPECTRUM= 1,    ///< Spectrum map
-    AF_COLORMAP_COLORS  = 2,    ///< Colors
+    AF_COLORMAP_SPECTRUM= 1,    ///< Spectrum map (390nm-830nm, in sRGB colorspace)
+    AF_COLORMAP_COLORS  = 2,    ///< Colors, aka. Rainbow
     AF_COLORMAP_RED     = 3,    ///< Red hue map
     AF_COLORMAP_MOOD    = 4,    ///< Mood map
     AF_COLORMAP_HEAT    = 5,    ///< Heat map
-    AF_COLORMAP_BLUE    = 6     ///< Blue hue map
+    AF_COLORMAP_BLUE    = 6,    ///< Blue hue map
+    AF_COLORMAP_INFERNO = 7,    ///< Perceptually uniform shades of black-red-yellow 
```

Comment:
Though github doesn't seem to display it, there is an additional space towards end all newly added enums. Please remove the it.

Reply:
@9prady9 Sorry about that.... I think that I got them all.